dockertest/images.py

```diff
@@ -430,6 +430,17 @@ def default_image(self):  # pylint: disable=R0201
         # DICLS may have overriden this method
         return DockerImage.full_name_from_defaults(defaults)
 
+    def default_image_exist_check(self):
+        """
+        Retrieve default image if it does not exist in
+         ``self.filter_list_full_name()``
+        :return: ``autotest.client.utils.CmdResult`` instance
+        """
+        img_list = self.list_imgs()
+        default_image = self.full_name_from_defaults()
+        if not self.filter_list_full_name(img_list, default_image):
+            return self.docker_cmd("pull %s" % default_image, self.timeout)
```

Comment:
I just wanted to wrap ``check`` and ``pull`` into one single function.
Looks like it's better to split info different part, so you mean we'd like to call ``docker pull`` in the test case and just return the image instance from this function?